<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="LAPTOP-JLSUG9BU" failures="1" tests="5" name="Test" time="9.237" errors="0" timestamp="2025-08-30T01:07:09 IST">
  <testcase classname="com.kart.automatedemokart.DemoKartTestSuite" name="tc001_addToCartWithoutLogin" time="0.23"/>
  <testcase classname="com.kart.automatedemokart.DemoKartTestSuite" name="tc002_loginWithValidCredentials" time="0.392"/>
  <testcase classname="com.kart.automatedemokart.DemoKartTestSuite" name="tc003_welcomeBackToastOnProductPage" time="1.127"/>
  <testcase classname="com.kart.automatedemokart.DemoKartTestSuite" name="tc004_addToCartAndValidateUrl" time="0.415">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [Phone 15 Pro Max has been added to your cart] but found [iPhone 15 Pro Max has been added to your cart]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [Phone 15 Pro Max has been added to your cart] but found [iPhone 15 Pro Max has been added to your cart]
at com.kart.automatedemokart.DemoKartTestSuite.tc004_addToCartAndValidateUrl(DemoKartTestSuite.java:63)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
... Removed 27 stack frames]]>
    </failure>
  </testcase> <!-- tc004_addToCartAndValidateUrl -->
  <testcase classname="com.kart.automatedemokart.DemoKartTestSuite" name="tc005_checkoutAndOrderConfirmation" time="4.471"/>
</testsuite> <!-- Test -->
